modules/github-typos.md

```diff
@@ -64,13 +64,13 @@ December 1989[[2]][5] by [Guido van Rossum][6] at [CWI][7]
 in [the Netherlands][8] as a **SUSSESSOR**/successor to [ABC][9] capable 
 of [exception handling][10] and **INTERAKTING**/interfacing with the 
 [Amoeba operating **SISTEM**][11].[[3]][12] Van Rossum 
-is Python's principal **AUTOR**, and his continuing 
+is Python's principal **AUTOR**/autor, and his continuing 
```

Comment:
Well, the original text contains not only formatting at specific points but also intended misspelling. Please, replace `autor` with relevant English word.